### DISCUSSION TITLE
patch plugin-commnet,not work , how to solve

### DISCUSSION THREAD

**Discussion by HFwas** [vuepress/ecosystem#101] [Q&A] [answered]
after started , console has a warning about plugin-comment,  i has find https://github.com/vuepress/ecosystem/commit/7ade07dd9fd4d51fb1753c6ef381cc4ad34aafed , so i update node_modules/@vuepress/plugin-comment/lib/client/pageview.js to generate patch , i delete node_modules , npm install , console still exist warning, 
![Uploading image.png…]()

![Uploading image.png…]()

**Comment by Mister-Hope** [comment, marked as answer]
This should be solved in latest v2
